src/interface/comm.rs

```diff
@@ -2,10 +2,13 @@
 // //
 // //
 
+use crate::interface;
 use std::mem::size_of;
 use std::sync::atomic::{AtomicBool, AtomicUsize, Ordering};
 use std::fs::read_to_string;
 use std::str::from_utf8;
+use std::os::unix::io::{AsRawFd, RawFd};
```

Comment:
I think we don't actually need to import this. Can't we just use ints?

Reply:
nice catch